.storybook/mswDecorator.js

```diff
@@ -0,0 +1,146 @@
+"use strict";
```

Comment:
I straight up copied this from the `dist` of `msw-storybook-addon`, after updating it to support MSW 2.0.

Really the only thing that's changes is the `require('msw/browser')` which used to be `require('msw')`.

Reply:
you could just copy/paste the typescript file instead, it would look cleaner and more understandable!

Reply:
It wouldn't work due to the way msw-storybook-addon is compiled differently than our project.